db/seeds/base/add_course_groups.rb

```diff
@@ -3,6 +3,6 @@
   specialist
   support
   ehco
-].each do |n|
-  CourseGroup.find_or_create_by!(name: n)
+].each do |name|
+  create(:course_group, name:)
```

Comment:
Think we need explicitly add FactoryBot here otherwise rails seeds task fails.

`FactoryBot.create(:course_group, name:)`

Reply:
Good spot, added 👍